io.github.thaunknown.miru.yaml

```diff
@@ -0,0 +1,53 @@
+app-id: io.github.thaunknown.miru
+base: org.electronjs.Electron2.BaseApp
+base-version: '22.08'
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: miru-run
+separate-locales: false
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --own-name=org.mpris.MediaPlayer2.chromium.instance3
+  - --socket=pulseaudio
+  - --filesystem=xdg-videos:rw
+  - --filesystem=xdg-run/discord-ipc-0:ro
+modules:
+  - name: miru
+    buildsystem: simple
+    build-commands:
+      - chmod +x miru.AppImage
+      - ./miru.AppImage --appimage-extract
+      - rm miru.AppImage
+
+      - cp squashfs-root/miru.desktop io.github.thaunknown.miru.desktop
+      - desktop-file-edit --set-icon io.github.thaunknown.miru --set-key Exec --set-value
+        'miru-run %u' io.github.thaunknown.miru.desktop
```

Comment:
you need to copy. Here just change it and then below install it from there. It's way simpler.